### ISSUE TITLE
Create a Reddit bot

### ISSUE THREAD

**Issue by Nutomic** [ipfspics/server#39]
That automatically mirrors images from imgur to ipfs.pics. And posts a link in the thread comments.

**Comment by leethax666** [comment]
that sounds expensive, but it might be good for controversial subreddits

**Comment by cloutier** [comment]
@Nutomic That is very expensive, I did back of the enveloppe calculation and it was over 30TB if we stored their picture in a very minimal quality. 

@noflag  A controversial subreddits could have it's own instance that they control, I could help if someone wants to do it.

**Comment by Nutomic** [comment]
Yeah it wouldn't work if you do it for all of reddit. But how about some specific subreddits, like /r/ipfs, /r/openbazaar, /r/btc, and similar ones. In any case I would start with only one, and see how it goes from there.

**Comment by cloutier** [comment]
Could be worthwhile, I am not sure though if it would be considered spam. Maybe publishing it in a way that just subreddit admins can enable it?
